packages/editor/src/pages/Auth/components/SomethingWentWrong.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { MessageBar, MessageBarType } from 'office-ui-fabric-react/lib/MessageBar';
+
+export default ({ additionalInfo }: { additionalInfo?: string }) => (
+  <MessageBar messageBarType={MessageBarType.severeWarning}>
+    Something went wrong. Please return to the code editor window and try again.
+    {additionalInfo ? (
+      <div>
+        <br />
```

Comment:
umm...... something doesn't seem right about this.... are you trying to add padding?